exercises/spiral-matrix/description.md

```diff
@@ -0,0 +1,22 @@
+Given a number, return a matrix of numbers in spiral order.
+
+The matrix should be filled with natural numbers, starting from 1
+in the top-left corner, increasing in a inward, clockwise spiral order,
+like these examples:
+
+- Spiral matrix of size 2
+
+    ```plain
```

Comment:
> Indenting should be unnecessary here since you are using the triple backticks?

The indenting was to nest the code block inside each item of the lists, but the result isn't as nice as I expected. I'll change it.

> where does the 2 come from here? The given matrix is 3x3!

Ops! I changed the examples and forgot to update it! 👍